CHANGELOG.md

```diff
@@ -34,6 +34,14 @@ Ref: https://keepachangelog.com/en/1.0.0/
 -->
 
 # Changelog
+## v25.17.0
+- f1b7756d fix: no routes for this trade (#480)
+- e6d44b60 perf: add load test for active orders (#472)
+- bc570d66 active order fault tolerance via "best-effort" boolean return  (#471)
+- 37f4424d add concurrency to the order book processing in active orders (#470)
```

Comment:
Let's remove all active orders query PRs from config. Instead, we will add one entry once we have long-term support.

We can add a single entry for new features to reduce unnecessary overhead from small iterations.

We should always think from the client's perspective and the amount of detail necessary for SQS users

```suggestion
```